### DISCUSSION TITLE
Resize from left or right direction indication to understand from where resize started.

### DISCUSSION THREAD

**Discussion by khanzzirfan** [samuelarbibe/dnd-timeline#53] [Q&A]
I would like to know if element is resizing from left or right.
In a video editor scenario , I want to know video is trimmed from left or right side of the video.

**Comment by samuelarbibe** [comment]
Hey!
element resizing can be done in any direction, and is dependent of the closest parent's `dir` value.  
Can you explain a bit more what you are trying to do?

**Reply by khanzzirfan** [reply]
Hello, Thanks for the reply. 
Let me reword my question here.

The callback function onResizeEnd, does not exactly tell if the resize was due to the left handle drag or  the right handle drag of an element. 

If I have a video element (let say). If i drag handle to resize the element from front (left side)   should indicate the video has to be trimmed from left.  And viz versa  if I do same thing from the right side handle resize. 

Hope this make bit more sense. or let me know, i'll create a video demo for you. Thanks again.

**Comment by khanzzirfan** [comment]
I think I found my answer. the resize event does have direction start and end.  Thank again.
```
type ResizeEndEvent = {
  active: Omit<Active, 'rect'>
  delta: {
    x: number
  }
  direction: DragDirection // 'start' | 'end'
}
```
